services/account-without-juggler/repositories/account/models/Account.ts

```diff
@@ -9,4 +9,19 @@ import {
 export class Account extends Entity {
   static definition = new ModelDefinition('Account', require('./account/model-definition').properties);
   static modelName = 'Account';
+
+  id: string;
+  customerNumber: string;
+  balance: number;
+  branch: string;
+  type: string;
+  avgBalance: number;
+  minimumBalance: number;
+
+  constructor(body?: Object) {
```

Comment:
I think you should be using `Partial<Account>` as the type of the `body` argument. That way typescript can offer auto-completion, and also limits the allowed `body` properties only the those that are defined on this model. See https://www.typescriptlang.org/docs/handbook/release-notes/typescript-2-1.html#mapped-types